src/components/tab-link/tab-link.tsx

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import style from './tab-link.module.css';
+import {
+  ComponentText,
+  TranslatedString,
+  useTranslation,
+} from '@atb/translations';
+import { Typo } from '../typography';
+import { AnimatePresence, motion } from 'framer-motion';
+
+type TabLinkProps = {
+  activePath: Route;
+};
+
+type LinkProps = {
```

Comment:
In other parts of the application, we name component prop types with the suffix `Props`. Since this type isn't a prop to the component, maybe we could rename it to `TabLink` or something similar? 
```suggestion
type TabLink = {
```

Reply:
Noted. Renamed the type now.